LEGO1/realtimeview.cpp

```diff
@@ -1,4 +1,11 @@
 #include "realtimeview.h"
+#include <math.h>
+
+// 0x10109598
+float g_userMaxLodPower;
```

Comment:
Is it correct for this to be uninitialized?

Reply:
Yep, that's correct. Wonder why they left it uninitialized. 
![image](https://github.com/isledecomp/isle/assets/106913236/5d51a1f2-633c-4a14-b045-ae64706e7856)